### DISCUSSION TITLE
MHI2_JP_VW48x_P0829 Region Conversion to EU

### DISCUSSION THREAD

**Discussion by veter4n** [Mr-MIBonk/M.I.B._More-Incredible-Bash#225] [General]
Hello everyone,

I succesfully installed m.i.b on my radio. Currently radio's region is Japan. I want to convert the train to EU. I tried to enable EU nav, but I got several errors like cannot open file /blah/blah/log. I guess this is because the script runs on none root privileges? I found the root password corresponding on my FW but I don't know where to insert it. If I select EU train conversion what should I know? What firmware should I prepare on a new SD card? Can someone assist my step by step this process please?

Thanks in advance.

**Comment by harman-f** [comment]
Please Share your log file from sd /backup/logs/

**Reply by veter4n** [reply]
The backup folder is absolutely empty and has no message.

![backup1](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/a9247ecc-ea16-4470-9b2a-43e76fe2dce2)
![backup2](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/5b49a380-276c-4265-9b88-da41b9d67d7f)

**Reply by harman-f** [reply]
Is your SD write protected?

**Reply by veter4n** [reply]
Indeed, my SD was protected. After I unlocked the SD, I successfully run enable EU NAV, and write var to eeprom rebooting twice the system without getting any errors. However, when I insert a second SD card and try to update nav database with correct EU maps, I still get the error that regions do not match 'Japan'' Europe'.
![IMG_20230525_000817](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/66592b76-5a66-4c8c-a87e-1241ec76de40)

**Reply by harman-f** [reply]
Log please

**Reply by veter4n** [reply]
```
 __  __     ___           ___  
|  \/  |   |_ _|         | _ ) 
| |\/| |    | |          | _ \ 
|_|  |_|ore|___|ncredible|___/ash - V3.3.0 "Bentley Edition - BYG24"
______________________________________________________________________

NOT FOR COMMERCIAL USE - IF YOU BOUGHT THIS YOU GOT RIPPED OFF

Logging_MU0829-MHI2_JP_VW48x_P0829-AW7-MON18.09.1700060038:
HW Number: 053 - PN: 5NA035050A
Variant: FM2-H-N-JP-VW-MQB
EEPROM Variant: MU-H-N-JP
Dataset Version: V03935262VW
Model ID: 027310 - Car Model: VW Golf (BQ1) 
VIN -> CAR: WVWZZZAUZJWXXXXXX - MIB: WVWZZZAUZJWXXXXXX

 ls --> total 2292
       8 drwxrwxrwx 10 root      root           4096 Jan 01 00:00 .
       8 drwxrwxrwx 10 root      root           4096 Jan 01 00:00 ..
       8 drwxrwxrwx  2 root      root           4096 May 21  2023 .System Volume Information
       2 -rwxrwxrwx  1 root      root            986 Jun 18  2022 .git-blame-ignore-revs
       1 -rwxrwxrwx  1 root      root            377 Jun 18  2022 .gitattributes
       1 -rwxrwxrwx  1 root      root            126 Jun 18  2022 .gitignore
    1703 -rwxrwxrwx  1 root      root         871810 Jun 18  2022 2022_06_MHI2_Password_List_V4.0.pdf
      81 -rwxrwxrwx  1 root      root          41255 Jun 18  2022 GEM.png
      30 -rwxrwxrwx  1 root      root          14939 Jun 18  2022 LICENSE
       8 drwxrwxrwx  4 root      root           4096 Jun 18  2022 Launcher
       1 -rwxrwxrwx  1 root      root             60 Jun 18  2022 M.I.B. - More Incredible Bash - Mibsolution Wiki.url
       0 -rwxrwxrwx  1 root      root              0 Jun 18  2022 NOT FOR COMMERCIAL USE - IF YOU BOUGHT THIS YOU GOT RIPPED OFF.txt
     284 -rwxrwxrwx  1 root      root         144962 Jun 18  2022 PATCH COMPATIBILITY TABLE.pdf
      10 -rwxrwxrwx  1 root      root           5084 Jun 18  2022 README.md
       1 -rwxrwxrwx  1 root      root             33 Jun 18  2022 VERSION
       1 -rwxrwxrwx  1 root      root            296 Jan 01  1980 WMPInfo.xml
       8 drwxrwxrwx  3 root      root           4096 Jun 18  2022 apps
       8 drwxrwxrwx  4 root      root           4096 Jan 01  1980 backup
      77 -rwxrwxrwx  1 root      root          39351 Jun 18  2022 bash.png
       8 drwxrwxrwx  2 root      root           4096 Jun 18  2022 config
       8 drwxrwxrwx  3 root      root           4096 Jun 18  2022 esd
       4 -rwxrwxrwx  1 root      root           1995 Jun 18  2022 metainfo2.txt
       8 drwxrwxrwx 10 root      root           4096 Jun 18  2022 mod
       8 drwxrwxrwx  3 root      root           4096 Jun 18  2022 patches
      16 -rwxrwxrwx  1 root      root           7989 Jun 18  2022 start

MMX-navon---->
Navigation is activated now
02731001ff00000051110001001038001f0100f601200400fe
Reboot Unit...
---- Read original Variant_1 from unit
Wait for DSI to connect ...
DSI Connected role1.DSIE2PAccess
0xba 	4d 55 2d 48 2d 4e 2d 4a 50 20 20 20 20 
DSI Disconnected
---- Clean Variant - 13 bytes 00
Wait for DSI to connect ...
DSI Connected role1.DSIE2PAccess
responseWriteE2P address 0xba status 0 len 0x0d
0xba 	00 00 00 00 00 00 00 00 00 00 00 00 00 
DSI Disconnected
---- Write Variant_1
Wait for DSI to connect ...
DSI Connected role1.DSIE2PAccess
responseWriteE2P address 0xba status 0 len 0x09
0xba 	4d 55 2d 48 2d 4e 2d 4a 50 
DSI Disconnected
---- Read Variant_1 back for log
Wait for DSI to connect ...
DSI Connected role1.DSIE2PAccess
0xba 	4d 55 2d 48 2d 4e 2d 4a 50 00 00 00 00 
DSI Disconnected

MMX-reboot---->
Reboot Unit...
```

**Reply by harman-f** [reply]
1. Get latest M.I.B - https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/archive/refs/heads/main.zip
2. Run: Main -> M.I.B. -> Advanced Settings -> EU Conversion

**Reply by veter4n** [reply]
Do I have to install it again via update software, or just copy paste it on the SD card?

**Reply by harman-f** [reply]
just copy new files to SD

**Reply by harman-f** [reply]
btw. could you get me a backup of the JP FW before you convert?
ZIP --> filehorst.de

**Reply by veter4n** [reply]
ofcourse! how can I backup the fw?

**Reply by veter4n** [reply]
I am currently backing up, all system, as soon as it's finished I will upload it. Regarding EU conversion, what specifically firmware should I have to a secondary SD card? There are several MHI2_VW* firmwares there. Also, by selecting enable EU conversion, what will be the process?

**Reply by harman-f** [reply]
/backup/ folder on SD

conversion target FW will be shown by M.I.B

**Reply by veter4n** [reply]
Backup has finished. So, on this screen I will select 1. EU conversion, the unit will reboot and the firmware that I will have to prepare will shown here next to Your current train?  Then I will download thw firmware and insert the SD card on slot 2. The I will run 3. Fix/screen bug  and right after this I will install EU firmware. Am I correct? ![IMG_20230525_231921.jpg](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/b6eb3eb2-fb60-4881-ae8d-d78059263f07)

**Reply by harman-f** [reply]
correct

**Reply by veter4n** [reply]
I successfully update the firmware. When I tried to update nav database, start update button is not clickable. What else do I have to know? Do I have to enable fec coding for eu nav? If yes how can I do this with m.i.b?
 Also why is supported list on fec empty on my unit? 

![IMG_20230526_102924.jpg](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/852857eb-df4a-4eb4-95c1-d7d2798c2066)

![IMG_20230526_102649.jpg](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/72556f0c-ff5f-4821-98ed-9fab169adeca)

![IMG_20230526_102659.jpg](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/119412216/035c06f7-b125-4a8c-8696-b9f2e57a4b14)

**Reply by harman-f** [reply]
Yes, you need to patch and EU FEC. 
share your log file.

**Comment by rsbob** [comment]
Patched also?

**Comment by Kraft89** [comment]
Hi Everyone,
Apologies if this post is in the wrong location but my issue is very similar to the above so I decided to add it here rather than create a new issue. 

I have an MHI2 unit from Japan and I wanted to convert it to EU region. I successfully installed M.I.B. and did a backup. I then carried out the EU conversion and all was going well until I got the black screen with no response after a reboot. As per the troubleshooting section, I got the unit back to life by removing the battery. 

When it came back on, the software train had changed from MHI2_JP_VW48x_P1001 to MHI2_ER_VWG13_P4521.

Does this mean that the conversion was successful or only partly completed? Would it be safe to proceed with the 'Fix Skin/Screen Bug'' step and then install the appropriate EU firmware (MHI2_ER_VWG13_K4525 according to mibhelper.com).

I have attached the backup log.

Thanks.

[MU1001-AW7-MON07.03.1900061614.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/15395367/MU1001-AW7-MON07.03.1900061614.log)
